pkg/controller/service/service_controller.go

```diff
@@ -374,21 +381,27 @@ func (s *ServiceController) syncLoadBalancerIfNeeded(service *v1.Service, key st
 	return op, nil
 }
 
+// Will this work for multiple resource partition clusters?
```

Comment:
this comment seems temporary dev purpose; may be able to remove?

Reply:
> this comment seems temporary dev purpose; may be able to remove?

I prefer leave it here since local cluster up does not start service controller. We don't have a way to test it locally. Will remove once it proves working

Reply:
Added TODO